hayeon/programmers-1/가운데-글자-가져오기.js

```diff
@@ -0,0 +1,30 @@
+function solution(s) {
+  var answer = "";
+  if (s.length % 2 === 0) {
+    answer = s[s.length / 2 - 1] + s[s.length / 2];
+  } else {
+    answer = s[Math.floor(s.length / 2)];
+  }
+  return answer;
+}
```

Comment:
```suggestion
function solution(s) {
  const midIdx = Math.floor(s.length / 2);
  let answer = "";
  
  if (s.length % 2 === 0) {
    answer = s[midIdx - 1] + s[midIdx];
  } else {
    answer = s[midIdx];
  }
  return answer;
}
```
- 고생하셨습니다 👍 풀이에 적어놓으신 것을 보니 엄청난 고민의 흔적이 느껴지네요 ㅋㅋ
- 우선 `var`로 변수 선언은 피해주세요!! var는 모던 자바스크립트에서 잘 사용하지 않습니다. 이유는 많은데요 그건 하연님께서 `var, let, const`의 차이를 학습하셨으면 좋겠습니다. (var 는 안쓰기로 약속합시다!!)
- 그리고 매번 s.length / 2를 하는 것보다 `하나의 변수`로 빼시면 훨씬 깔끔하지 않을까 생각이드네요 👍
이부분은 연재님 코드를 참고해보세요 :)

Reply:
넵 !! 감사합니다 한번 확인하고 다시 풀어보겠습니다!!!!!!! 👏